control/grpc.py

```diff
@@ -132,7 +133,14 @@ def create_bdev(self, request, context=None):
 
         return pb2.bdev(bdev_name=bdev_name, status=True)
 
-    def delete_bdev(self, request, context=None):
+    def create_bdev(self, request, context=None):
+        if context:
```

Comment:
@baum when we come from update() we already have the lock. This is when context is None. Originally I added a code to raise an exception in case we got there with holding the lock. So I could make sure it works OK.

Reply:
where in update()? could you point to the code please?

Reply:
@baum it's in gateway_rpc_caller() which we get to from update().